test/uploader.test.ts

```diff
@@ -83,16 +84,22 @@ beforeAll(() => {
 
 describe("Uploader", () => {
   it("Should support uploading files", () =>
-    new Promise((done) => {
-      const uploader = new Uploader(session, file, { onComplete: done });
+    new Promise<void>((resolve) => {
+      const onComplete = vitest.fn();
+      const uploader = new Uploader(session, file, { onComplete });
       uploader.start();
+      onComplete.mockImplementation(() => {
+        expect(onComplete).toHaveBeenCalledOnce();
+        resolve();
+      });
```

Comment:
eslint started complaining about that tests need to include an assertion (expect), so redid this test a bit to include it